test/train-sets/ref/squarecb_load.stderr

```diff
@@ -10,15 +10,15 @@ power_t = 0.5
 Enabled reductions: gd, scorer-identity, csoaa_ldf-rank, cb_adf, cb_explore_adf_squarecb, shared_feature_merger
 Input label = cb
 Output pred = action_probs
-average  since         example        example        current        current  current 
-loss     last          counter         weight          label        predict features 
-0.000000 0.000000            1            1.0       4:0:0.14     0:0.911854       42 
-0.000000 0.000000            2            2.0       0:0:0.14     0:0.818539       42 
-0.000000 0.000000            4            4.0       1:0:0.14     2:0.893115       42 
-0.000000 0.000000            8            8.0       3:0:0.14     2:0.693382       42 
-0.000000 0.000000           16           16.0       3:0:0.14     2:0.693382       42 
-0.000000 0.000000           32           32.0       3:0:0.14     2:0.893115       42 
--0.56272 -1.12544           64           64.0       0:0:0.77     0:0.818539       42 
+average  since         example        example        current        current  current
+loss     last          counter         weight          label        predict features
+0.000000 0.000000            1            1.0       4:0:0.14         0:0.91       42
```

Comment:
what file change fixed the missing spaces on current predict column?

Reply:
It was actually fixed in the original PR [here](https://github.com/VowpalWabbit/vowpal_wabbit/pull/3741/commits/9bdeb79806cc4d0dbc904e8ec74947c91829f9e5). But because we trim the diff it wasnt triggering failures.